pkgs/development/ocaml-modules/junit/default.nix

```diff
@@ -0,0 +1,17 @@
+{ lib, fetchFromGitHub, buildDunePackage, ptime, tyxml }:
+
+let
+  shared = (import ./shared.nix) { inherit lib fetchFromGitHub; };
+in
+  buildDunePackage rec {
+    pname = "junit";
+    inherit (shared) src version meta;
+    useDune2 = true;
```

Comment:
The `useDune2` attribute belongs to `shared`.

Reply:
Done